src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -579,6 +580,10 @@ XElement CreateApplicationElement (XElement manifest, string applicationClass, L
 				Assemblies.SelectMany (path => MetaDataAttribute.FromCustomAttributeProvider (Resolver.GetAssembly (path), cache))
 					.Where (attr => attr != null)
 					.ToList ();
+			var properties =
+				Assemblies.SelectMany (path => PropertyAttribute.FromCustomAttributeProvider (Resolver.GetAssembly (path), cache))
+					.Where (attr => attr != null)
+					.ToList ();
```

Comment:
This shouldn't be fixed here, but I wonder if the N `Assemblies.SelectMany()` calls could be a performance issue.